<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoreUI</class>
 <widget class="QMainWindow" name="CoreUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenCV Face Recognition System - Core</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="systemSettingsGroupBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>321</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>系统设置</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QGroupBox" name="functionGroupBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>功能开关</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="faceTrackerCheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>人脸跟踪</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="faceRecognizerCheckBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>数据库未完成初始化，请检查</string>
         </property>
         <property name="text">
          <string>人脸识别</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="webcamGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>图像捕获</string>
     </property>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>21</y>
        <width>134</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="webcamVerticalLayout">
       <item>
        <widget class="QCheckBox" name="useExternalCameraCheckBox">
         <property name="text">
          <string>使用外接摄像头</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startWebcamButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>打开摄像头</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="dbGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>数据库</string>
     </property>
     <widget class="QPushButton" name="initDbButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>检查数据库状态</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="debugGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>301</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Debug设置</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QCheckBox" name="debugCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>调试模式</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="debugGridLayout">
       <item row="0" column="1">
        <widget class="QSlider" name="confidenceThresholdSlider">
         <property name="maximum">
          <number>80</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="confidenceThresholdLabel">
         <property name="text">
          <string>置信度阈值：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="confidenceTipLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>置信度阈值越小，精确度要求越高。开启调试模式后将实时输出人脸识别反馈。</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="realTimeCaptureGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>641</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>摄像头实时画面</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="realTimeCaptureLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>640</width>
       <height>500</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>&lt;font color=red&gt;摄像头未开启&lt;/font&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="aboutLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>530</y>
       <width>541</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>基于OpenCV的视频人脸识别</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="logGroupBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>360</y>
      <width>321</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>系统日志</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QTextEdit" name="logTextEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>321</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
